README.md

```diff
@@ -34,76 +34,126 @@ DeepRank2 extensive documentation can be found [here](https://deeprank2.rtfd.io/
 
 ## Table of contents
 
-- [Deeprank2](#deeprank2)
+- [DeepRank2](#deeprank2)
   - [Overview](#overview)
   - [Table of contents](#table-of-contents)
-  - [Installation](#installation)
-    - [Dependencies](#dependencies)
-  - [Deeprank2 Package](#deeprank2-package)
-    - [Test installation](#test-installation)
-    - [Contributing](#contributing)
-    - [Data generation](#data-generation)
-    - [Datasets](#datasets)
-      - [GraphDataset](#graphdataset)
-      - [GridDataset](#griddataset)
-    - [Training](#training)
+  - [Installations](#installations)
+    - [Containerized Installation](#containerized-installation)
+    - [Local/remote installation](#localremote-installation)
+      - [Non-pythonic dependencies](#non-pythonic-dependencies)
+      - [Pythonic dependencies](#pythonic-dependencies)
+      - [Deeprank2 Package](#deeprank2-package)
+      - [Test installation](#test-installation)
+  - [Contributing](#contributing)
+  - [Data generation](#data-generation)
+  - [Datasets](#datasets)
+    - [GraphDataset](#graphdataset)
+    - [GridDataset](#griddataset)
+  - [Training](#training)
```

Comment:
When I save the file in vscode, it does it automatically for me. I think it's because I have [this extension](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one)